templates/workspace_services/guacamole/user_resources/guacamole-azure-import-reviewvm/windowsvm.diff

```diff
@@ -1,38 +1,31 @@
-diff --git terraform/variables.tf ../guacamole-azure-windowsvm/terraform/variables.tf
```

Comment:
This is a bit hard to read in the diff as-is, I suggest go to 3 dot menu -> View file